packages/components/src/molecules/SearchHistory/SearchHistory.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { HTMLAttributes } from 'react'
+import { List, Button } from '../..'
+
+export interface SearchHistoryProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  /**
+   * Defines the section's title.
```

Comment:
I think this could be made clearer. How about

```suggestion
   * Title attribute for the <section> tag rendered by this component.
```